src/tests/ftest/server/daos_server_dump.yaml

```diff
@@ -8,7 +8,7 @@ hosts:
     - server-D
     - server-E
     - server-F
-timeout: 360
+timeout: 140
```

Comment:
140s is fine as it's a relatively small value, but for what it's worth, you can see how long a test takes in the output of the test run:

```
[2021-04-22T06:24:42.990Z]  (1/1) ./server/daos_server_dump.py:DaosServerDumpTest.test_daos_server_dump_basic;hosts-servers-4038:  PASS (76.00 s)
```